keras_trainer/callbacks.py

```diff
@@ -0,0 +1,49 @@
+import os
+import numpy as np
+
+from keras.callbacks import Callback
+from sklearn.metrics import recall_score
+
+
+class SensitivityCallback(Callback):
+    def __init__(self, validation_data_generator, output_model_dir, batch_size=10):
+        super(SensitivityCallback, self).__init__()
+        self.validation_data_generator = validation_data_generator
+        self.batch_size = batch_size
+        self.output_model_dir = os.path.join(output_model_dir, 'model_max_sensitivity.hdf5')
```

Comment:
More general to set `output_model_filename` to be the path and the filename? Rather than hardcoding a model name here?

Reply:
This is for re using https://github.com/triagemd/keras-trainer/blob/add_sensitivity_callback/keras_trainer/trainer.py#L26
and model name follows same structure as https://github.com/triagemd/keras-trainer/blob/add_sensitivity_callback/keras_trainer/trainer.py#L235